src/image.tsx

```diff
@@ -8,6 +8,32 @@ import getConfig from './utils/getConfig'
 
 const config = getConfig()
 
+const defaultImageParser = (src: string) => {
```

Comment:
It would be better if the return value had `ParsedImageInfo`!

Reply:
Agreed - I'll add the function signature/type